app/views/providers/start/index.html.erb

```diff
@@ -14,9 +14,6 @@
 
       <p class="govuk-body"><%= t(".dont_use_service.list_title") %></p>
       <ul class="govuk-list govuk-list--bullet">
-        <% if !Setting.partner_means_assessment? %>
-          <li><%= t(".dont_use_service.partner_list_item") %></li>
```

Comment:
Can key value now be removed from https://github.com/ministryofjustice/laa-apply-for-legal-aid/blob/e8466643eaf1e973f4f9829d801ac0c804bb9bcb/config/locales/en/providers.yml#L1558

Reply:
👍